.github/commands.json

```diff
@@ -322,5 +322,13 @@
     "addToProject": {
       "url": "https://github.com/orgs/grafana/projects/78"
     }
+  },
+  {
+    "type": "label",
+    "name": "area/backend",
```

Comment:
So any backend ticket will end up in the backend board and if it's not backend platform responsibility, the backend platform issue triage should delegate it to the right team?

Reply:
Good point, maybe I was too fast to open a a PR. I'll check the past issues that are labeled `area/backend` - maybe I'll close it afterwards or change the label(s). Don't want backend platform to route the issues, just wanted to make sure our issues are on the board.